packages/wrangler/src/api/index.ts

```diff
@@ -1,2 +1,3 @@
 export { unstable_dev } from "./dev";
 export type { UnstableDevWorker, UnstableDevOptions } from "./dev";
+export { unstable_pages } from "./pages";
```

Comment:
You might want to export the types for `unstable_pages` too?

Reply:
If you don't mind, I'd like to hold off on that rabbit hole. There's a lot of types involved and I'd like to move more code around to export all of the relevant bits. If a consumer needs the type of the argument to `publish` or the return type, then they can easily infer it:

```tsx
type PublishOptions = typeof unstable_pages.publish extends (options: infer P) => any ? P : never
type PublishResult = typeof unstable_pages.publish extends (options: any) => infer P ? P : never
```

I - being the only consumer of this code so far - don't actually need that yet :)